tests/integration-tests/tests/pyxis/test_pyxis/test_pyxis/pcluster.config.yaml

```diff
@@ -0,0 +1,22 @@
+Image:
+Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  SharedStorageType: Efs
+Scheduling:
+  Scheduler: {{ scheduler }}
+  SlurmQueues:
+    - Name: queue-0
+      ComputeResources:
+        - Name: compute-resource-0
+          Instances:
+            - InstanceType: t3.small
+          MinCount: 0
+          MaxCount: 2000
```

Comment:
Let's inject the value from the test so that we have a single variable that can be used in the submission commands as well